src/pages/AddLiquidity/index.tsx

```diff
@@ -363,6 +372,21 @@ export default function AddLiquidity({
     [currencyIdA, currencyIdB, history, onLeftRangeInput, onRightRangeInput]
   )
 
+  const handleBothCurrenciesSelect = useCallback(() => {
+    let _showL2Suggestion = false
+    if (fiveEthUsdcValue) {
+      // Only suggest L2 to users who would like to add liquidity that is worth less than 5 ETH
+      const currencyAusdcValue = usdcValues[Field.CURRENCY_A]
+      const currencyBusdcValue = usdcValues[Field.CURRENCY_B]
+      if (currencyAusdcValue && currencyBusdcValue) {
+        _showL2Suggestion = currencyAusdcValue.add(currencyBusdcValue).lessThan(fiveEthUsdcValue)
+      }
+    }
+    setShowL2Suggestion(_showL2Suggestion)
+  }, [usdcValues, fiveEthUsdcValue])
```

Comment:
@willhennessy thoughts on this? (only shows the L2 prompt to people adding <5 eth liquidity)

Reply:
@fionnachan I think this should be removed

Reply:
Done